game-core/src/main/java/org/triplea/game/common/ChatConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package org.triplea.game.common;
```

Comment:
> A "chat" package would be pretty reasonable instead of "common".

I'm assuming the `common` here refers to the fact that this code is shared between both client and server, so as to distinguish it from packages such as `org.triplea.game.client` and `org.triplea.game.server`, both of which exist today.  In that case, I think `common` is appropriate.  However, if `common` is too vague, then we should decide upon a better name to indicate code that's neither specific to the client nor the server, as we already use the client/server/common convention for the lobby.

Alternatively, we could view anything not under `o.t.game.client` or `o.t.game.server` as being implicitly "common".  For example, `o.t.game.chat` would contain chat code shared between client and server, while `o.t.game.server.chat` would contain server-specific chat code.  I'm okay with that, but I do like the symmetry of package depth the client/server/common convention offers.

<hr>

However, I do agree that this type should be in a `chat` subpackage, whether under `o.t.game`, `o.t.game.common`, or something else entirely.